regulations/convert.py

```diff
@@ -0,0 +1,150 @@
+"""Build index of document URLs from a Regulations.gov bulk download file"""
+
+import argparse
+import html
+import itertools
+import json
+import os
+import subprocess
+
+import requests
+from bs4 import BeautifulSoup
+from tqdm.auto import tqdm
+
+from licensed_pile import logs
+
+
+def parse_args():
+    parser = argparse.ArgumentParser("Regulations.gov index builder")
+    parser.add_argument(
+        "--input-dir", required=True, help="Path to directory containing files"
+    )
+    parser.add_argument("--output-dir", required=True, help="Path to output directory")
+    parser.add_argument(
+        "--years",
+        nargs="+",
+        default=list(map(str, range(2000, 2024))),
+        help="Years of data to process",
+    )
+    parser.add_argument(
+        "--agencies",
+        nargs="+",
+        default=[
+            "bis",
+            "dot",
+            "epa",
+            "faa",
+            "fda",
+            "fema",
+            "ferc",
+            "fmcsa",
+            "fra",
+            "nhtsa",
+            "osha",
+            "phmsa",
+            "sec",
+            "uscg",
+        ],
+        help="Agencies to process",
+    )
+    parser.add_argument(
+        "--file-types",
+        nargs="+",
+        default=[".txt", ".htm", ".doc", ".docx"],
+        help="File types to convert to .txt",
+    )
+    parser.add_argument(
+        "--overwrite",
+        action="store_true",
+        default=False,
+        help="Overwrite existing files in output directory",
+    )
+    return parser.parse_args()
+
+
+def convert_htm(input_path, output_path):
```

Comment:
prefer naming this `convert_html`, `htm` is super legacy from when windows couldn't have an extension with > 3 chars lol